### DISCUSSION TITLE
Writing more tests

### DISCUSSION THREAD

**Discussion by Agrover112** [guofei9987/scikit-opt#151] [General]
As we noted in our last Pull Request which was merged seems like this repository has very few tests.
So I was thinking of maybe moving some of the print statements and convert them into unit-tets?

**Comment by guofei9987** [comment]
Yes, tests are important for objective functions.

**Reply by Agrover112** [reply]
Ok, I will write some unit-tests. You could also add tests for ajy edge cases id you find them!(Will raise a PR)

**Comment by Agrover112** [comment]
@guofei9987  Pull Request raise #153 !
